web/src/HUD.tsx

```diff
@@ -188,6 +194,18 @@ class HUD extends Component<HudProps, HudState> {
       .catch(err => console.error(err))
   }
 
+  handleSetHighlight(highlight: SnapshotHiglight) {
+    this.setState({
```

Comment:
does this get properly cleared when you switch views? seems like you could have a highlight associated with the wrong resource

Reply:
We need to look in to this

Reply:
It does get cleared! When you switch logs the document selection is "collapsed", at which point we clear the highlight from the state.